rotkehlchen/tests/unit/decoders/test_uniswapv2.py

```diff
@@ -747,3 +748,196 @@ def test_uniswap_v2_remove_liquidity(database, ethereum_inquirer, eth_transactio
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[ADDY_4]])
+def test_uniswap_v2_swap_events_order(
+        database,
+        ethereum_inquirer,
+        eth_transactions,
+        ethereum_accounts,
+):
+    """Check that the order of swap events are consecutive."""
+    tx_hash = '0x75b4d35b14f435c60e0fb9a82a70457a213c91ef829c034c743c66bec2dcd40f'
```

Comment:
Can you state that the transaction hash here has been randomly selected to protect the transaction of the user? We will at some point look at this transaction, see that it doesn't match and wonder why